### PR TITLE
Add common support for API in packages

### PR DESCRIPTION
**Pull request by pdbreen**

### CODE REVIEW

File: src/TipoffPackage.php [hunk 1]
```diff
@@ -58,6 +58,24 @@ public function __construct(Package $package)
...
+    }
+
+    public function hasRoutes(...$routeFileNames): Package
+    {
+        if (config('tipoff.api.enabled')) {
+            return parent::hasRoutes(...$routeFileNames);
+        }
+
+        return $this;
+    }
```
Comment (drewroberts) [inline src/TipoffPackage.php]: Should these be `hasApiRoute` and `hasApiRoutes` to distinguish from the other routes the package may provide?

Comment (pdbreen) [reply src/TipoffPackage.php]: Sure - we can do that.  I'll add in config and methods for web routes at the same time.  One sec.

Comment (drewroberts) [reply src/TipoffPackage.php]: Thanks!

**Review by drewroberts — APPROVED** [review]
This is great! Thank you! 🚜

**Pull request merged**